bin/ember-template-lint.js

```diff
@@ -194,7 +194,7 @@ function parseArgv(_argv) {
       },
       'clean-todo': {
         describe: 'Remove expired and invalid todo files',
-        default: false,
+        default: true,
```

Comment:
@scalvert - What do you think about making the default here something like:

```js
const ci = require('ci-info')

const shouldCleanTodos = !ci.isCI;
```

Then you'd do this:

```suggestion
        default: shouldCleanTodos,
```

Reply:
Ya I think this is good. This handles the case we want WRT CI. I'll make that change.

Reply:
Done.